feature-libs/cart/base/core/http-interceptors/handlers/bad-cost-center-request.handler.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from "@angular/core";
+import { HttpErrorResponse, HttpRequest } from "@angular/common/http";
+import { isEntityValidationError} from "../../utils/utils";
+import {
+  ErrorModel,
+  GlobalMessageType,
+  HttpErrorHandler,
+  HttpResponseStatus,
+  Priority,
+} from '@spartacus/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class BadCostCenterRequestHandler extends HttpErrorHandler {
+  responseStatus = HttpResponseStatus.BAD_REQUEST;
+
+  getPriority(): Priority {
+    return Priority.NORMAL;
+  }
+
+  hasMatch(errorResponse: HttpErrorResponse): boolean {
+    return (
+      super.hasMatch(errorResponse) &&
+      this.getErrors(errorResponse).some(isEntityValidationError) &&
+      this.isCostCenterRequest(errorResponse)
+    );
+  }
+
+  handleError(_request: HttpRequest<any>, response: HttpErrorResponse): void {
+    this.getErrors(response)
+      .filter((e) => isEntityValidationError(e))
+      .forEach(() => {
```

Comment:
Do we really want to display multiple occurences of the same message for the same backend call?